src/dewey/entity.clj

```diff
@@ -12,15 +12,15 @@
 (defn- lookup-base
   [irods path]
   (try
-    (.getCollectionAndDataObjectListingEntryAtGivenAbsolutePath (:lister irods) path)
+    (.getCollectionAndDataObjectListingEntryAtGivenAbsolutePath (:lister @irods) path)
```

Comment:
The changes within this file are all about compatibility with the "lazy" context map and are about adding derefs when the irods connection is actually used. The hope is that these dereferences will happen at the last possible second before needed. An alternative approach might be to push the derefs up within the curation namespace instead, doing it as early as we know that irods will be needed in the handling of the message (so, at the beginning for most messages, and after hitting elasticsearch for data-object.metadata.add). In the latter approach, this namespace would probably assume it has a real context map and not a delay.

Let me know your thoughts on these different approaches (connect to irods as soon as we know it's needed vs. as close as possible to where it's actually used)